src/components/common/entityData/EntityDataColumns/cmp.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { memo } from 'react'
+import { EntityDataColumnsProps } from './types'
+
+export const EntityDataColumns = ({
+  leftColumnElements,
+  rightColumnElements,
+}: EntityDataColumnsProps) => {
+  const renderElements = (elements: React.ReactNode[]) =>
```

Comment:
use useCallback here or create a subcomponent, otherwise the list will be re-rendered each time